### DISCUSSION TITLE
RM-LXC bombardiert den lokalen DNS-Server mit Anfragen nach "localhost", da läuft irgendwas nicht korrekt

### DISCUSSION THREAD

**Discussion by Baxxy13** [OpenCCU/OpenCCU#2782] [🇩🇪 Fragen&Antworten] [answered]
Mir ist aufgefallen das der LXC im Sekundentakt meinen lokalen DNS-Server (Adguard-Home) nach "localhost" befragt.
Das macht keins der anderen RM-Systeme hier.
Bei der Recherche ist mir dann aufgefallen das der LXC keine `/var/etc/hosts` erzeugt, was womöglich mit den DNS-Anfragen zusammenhängt.
Hab mir auch mal die für's Netzwerk zuständige `/etc/network/if-up.d/eQ3StartNetwork` angesehen, die ist ja identisch zu den "nicht-LXC-Versionen". Konnte aber erstmal nicht herausfinden warum die `/var/etc/hosts` nicht durch `/etc/network/if-up.d/eQ3StartNetwork` erzeugt wird.

Vielleicht liege ich ja auch falsch, daher sind hilfreiche Hinweise willkommen.

Addendum: Hatte zum testen mal den Historian im LXC installiert, da gab es java-Fehler die irgendwas mit der Host-Namensauflösung zu tun hatten.
![RM-LXC_Adguard_DNS](https://github.com/jens-maus/RaspberryMatic/assets/54898249/e88682d8-ab0e-4765-8b63-e51bfac2df8d)

**Comment by jens-maus** [comment]
Hast du die Hosts Datei mal einfach innerhalb des LXC angelegt und geschaut ob sich das ganze dann beruhigt?

**Reply by Baxxy13** [reply]
Ja, hatte ich probiert. Hatte nichts gebracht.
Auch ein `/etc/init.d/S40network restart` während die `hosts` vorhanden ist half nichts. 
Da die `hosts`-Datei im `/var` liegt kann ich auch nicht rebooten da sie dann weg ist.

**Reply by Baxxy13** [reply, marked as answer]
Das Problem lag in der Tat bei der fehlenden `hosts` - Datei.
Mit dem heutigen Nightly `RaspberryMatic-3.77.4.20240621-f97433` ist diese korrekt vorhanden und mein DNS-Server wird nicht mehr mit `localhost` - Anfagen belästigt.

Sehr gut. 👍 

Damit bin ich aktuell mit "LXC-Problemen" erstmal durch, abgesehen von #2763.
